generate.py

```diff
@@ -624,6 +635,12 @@ def callback(x):
     parser.add_argument(
         "--device", type=str, default=default_device, help="Device to use"
     )
+    parser.add_argument(
+        '--ignore_gist_while_generation',
```

Comment:
Is there ever a scenario where we would want this to be False?

If we always want to ignore it, we can just remove this as a flag and avoid having to pass the boolean flag around the file